src/olympia/addons/indexers.py

```diff
@@ -343,7 +346,8 @@ def extract_document(cls, obj):
 
         # We can use all_previews because the indexing code goes through the
         # transformer that sets it.
-        data['previews'] = [{'id': preview.id, 'modified': preview.modified}
+        data['previews'] = [{'id': preview.id, 'modified': preview.modified,
+                             'sizes': json.dumps(preview.sizes)}
```

Comment:
As discussed in IRC, I'm not a fan of saving this as plain text instead of saving this as an actual JSON document. I'm not sure how that can be handled via the index definition but maybe something to look at.

Reply:
https://github.com/mozilla/addons-server/pull/6803/commits/6f07c8f74c0c7a85f1852605461a0455335ff3dd